src/main/modules/oidc/index.ts

```diff
@@ -47,8 +49,24 @@ export class OidcMiddleware {
       req.session.save(() => res.redirect('/'));
     }));
 
-    app.use((req: Request, res: Response, next: NextFunction) => {
+    app.use(async (req: Request, res: Response, next: NextFunction) => {
       if (req.session?.user) {
+        const user = req.session.user;
+        req.scope = req.app.locals.container.createScope();
+        req.scope?.register({
+          axios: asValue(Axios.create({
+            baseURL: config.get('services.cos.baseURL'),
+            headers: {
+              Authorization: 'Bearer ' + user.id_token
+            }
+          })),
+          api: asClass(COSApi)
+        });
+
+        if (!req.session.caseCreated) {
```

Comment:
do we want to try a GET /case before creating one?

Reply:
I wasnt sure the endpoint was created yet so i left it like that, ill speak to sarah to see.